testing/testing_page.html

```diff
@@ -798,35 +798,27 @@ <h4 class="modal-title" id="myModalLabel">Modal Title</h4>
         <span class="pficon pficon-ok"></span>
         <strong>Retirement date set to 12/31/19 15:55 UTC</strong>
       </div>
-    </div>
-    <div class="flash_text_div">
```

Comment:
Does this change match the reference implementation of PF3, or something from MIQ?

Reply:
Got an answer offline, its a bit of both, but inline with reference code from Patternfly docs.